### PR TITLE
Added new Range Ring tools to Visibility and Range Tools toolbox

### PR DESCRIPTION
**Pull request by mfunk**

### CODE REVIEW

**Comment by mfunk** [comment]
This pull adds the new range ring tools to the Visibility and Range Toolbox, adds the scripts, layer files, and toolboxes. Unit tests will be added next in a separate pull request.
@pshowalter @topowright @lfunkhouser

File: visibility/toolboxes/scripts/RangeRingUtils.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    return rangeRingsFromList(centerFC, rangeList, distanceUnits, numRadials, outputRingFeatures, outputRadialFeatures, sr)
+
+def rangeRingsFromInterval(centerFC, numRings, distBetween, distanceUnits, numRadials, outputRingFeatures, outputRadialFeatures, sr):
+    ''' Classic range rings from number of rings, and distance between rings  '''
+    rangeList = [x * distBetween for x in range(1, numRings + 1)]
+    return rangeRingsFromList(centerFC, rangeList, distanceUnits, numRadials, outputRingFeatures, outputRadialFeatures, sr)
+
+
+
+class RingMaker:
```
Comment (csmoore) [inline visibility/toolboxes/scripts/RangeRingUtils.py]: Is there any reason why RingMaker can't be in its own file?

**Comment by csmoore** [comment]
Sorry I never got around to testing on Pro as requested - but I review the code and the RingMaker class and tests looks like a great addition. A couple minor comments noted.

File: visibility/toolboxes/scripts/RangeRingUtils.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    '''
+    Core class for making range rings.
+    '''
+
+    def __init__(self, center, inputRangeList, distanceUnits, sr):
+        ''' initialize rings '''
+
+        self.deleteme = []
```
Comment (csmoore) [inline visibility/toolboxes/scripts/RangeRingUtils.py]: Every time I see this in the code it confuses me a bit (and I have to track down what it does) - so perhaps a more descriptive name would help like "tempDatasetsToDelete"

Comment (mfunk) [inline visibility/toolboxes/scripts/RangeRingUtils.py]: You mean **self.deleteme = []**? OK. Guess it's just me being a smartass. :)

Comment (csmoore) [inline visibility/toolboxes/scripts/RangeRingUtils.py]: Yes, everything I see it I think it is another Python Easter egg like: "import this" - http://legacy.python.org/dev/peps/pep-0020/

**Pull request merged**